hw/mcu/dialog/da1469x/src/hal_gpio.c

```diff
@@ -473,6 +474,10 @@ mcu_gpio_enter_sleep(void)
         return;
     }
 
+    /* Save pins states and their latched values */
```

Comment:
@JustineKH Are interrupts already disabled when `mcu_gpio_enter_sleep` gets called? If not,  an interrupt could change pin levels before entering sleep.

Reply:
the whole sleep code is run with interrupts disabled since it's called from 'os_tick_idle`